src/events/ready.ts

```diff
@@ -5,8 +5,10 @@ import banCommand from '@/commands/ban';
 import kickCommand from '@/commands/kick';
 import settingsCommand from '@/commands/settings';
 import warnCommand from '@/commands/warn';
+import { checkMatchmakingThreads } from '@/matchmaking-threads';
 import type { Client, Collection } from 'discord.js';
 import type { ClientCommand } from '@/types';
+import { scheduleJob } from 'node-schedule';
```

Comment:
We have a preferred order when it comes to imports, where `type` imports always come last

The order we use is:

- Native modules (which also use the `node:` prefix)
- 3rd party modules
- local modules (which always use the full import alias path, regardless of where the import is at)
- types (which always use `import type` style syntax)
- data imports (JSON, etc.)

Reply:
Any way to lint/automate this for the future?

Reply:
Not that I'm aware of, though I have not done a ton of research into that